fissix/tests/__init__.py

```diff
@@ -1,9 +0,0 @@
-# Author: Collin Winter
-
-import os
-
-from test.support import load_package_tests
-
-
-def load_tests(*args):
-    return load_package_tests(os.path.dirname(__file__), *args)
```

Comment:
I couldn't work out how to get this to work, so I used pytest to collect instead

```
Traceback (most recent call last):
  File "/home/graingert/projects/fissix/fissix/tests/__init__.py", line 9, in load_tests
    return load_package_tests(os.path.dirname(__file__), *args)
  File "/usr/lib/python3.8/test/support/__init__.py", line 243, in load_package_tests
    package_tests = loader.discover(start_dir=pkg_dir,
AssertionError: Path must be within the project
```